src/molecule/util.py

```diff
@@ -96,7 +96,10 @@ def sysexit(code: int = 1) -> NoReturn:
 
 
 def sysexit_with_message(
-    msg: str, code: int = 1, detail: Optional[MutableMapping] = None
+    msg: str,
+    code: int = 1,
+    detail: Optional[MutableMapping] = None,
+    warns: Optional[List[Warning]] = None,
```

Comment:
I think list is better because I suspect we will add to it later, when we find other places that can raise warnings. Having an iterable does not allow us to add to it.